about_us.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: About Us
+nav_order: 10
+---
+
+# About Us
+{: .no_toc }
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+<img src="../images/medik8s-logo.png" alt="nmo-icon" width="200" style="margin-left:auto; margin-right:auto; display:block"/>
+
+## Who are we?
+
+We, Medik8s team, are four Red Hatters with a strong passion to High Availabitlity (HA) solutions in the Kubernetes world.
+
+## What we do?
+
+We develop Kubernetes Open Source Operators using [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/) that provide maintenance support, automatic node remediation and high availability for singleton workloads:
+
+- [Node Healthcheck Operator (NHC)](failure_detection) - Detecting Node Failures, triggering remediation, which is performed by other operators like Self Node Remediation or MachineDeletion.
+- [Self Node Remediation (SNR)](/remediation/self-node-remediation/self-node-remediation) - Remediates nodes by rebooting, without needing a management interface like e.g. IPMI, or a node / machine provisioning API. Works standalone, and / or with NHC.
+- [Node Maintenance Operator (NMO)](maintenance-node) - Declarative node cordoning and draining prior to harmful decisions.
+
+## What we plan to do?
+
+We plan to improve the exiting operators by adding Master (control plane) fencing capabilites to SNR and NHC, which currently **only** work for worker nodes.
+Furthermore, we have few work in progress operators:
+
+- [Machine Deletion](/remediation/machine-deletion/machine-deletion) - Remediates nodes by deleting the associated OpenShift machine. Triggered by NHC.
+- [HA-SNO](https://github.com/medik8s/ha-sno) - High Availability Single Node Openshift.
```

Comment:
Might want to mention fence agents as well - https://github.com/mshitrit/fence-agents-remediation